edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -29,21 +32,6 @@ include::modules/ztp-image-based-upgrade-prep.adoc[leveloffset=+1]
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR]
 
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR]
-
-include::modules/ztp-image-based-upgrade-upgrade.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-rollback_ztp-gitops[Moving to the Rollback stage of the image-based upgrade with {lcao} and {ztp}]
-
-* xref:../../edge_computing/cnf-talm-for-cluster-upgrades.adoc#talo-policies-concept_cnf-topology-aware-lifecycle-manager[Update policies on managed clusters]
-
-include::modules/ztp-image-based-upgrade-rollback.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-scenario-3-recovering-expired-certs_dr-recovering-expired-certs[Recovering from expired control plane certificates]
+////
 
 include::modules/cnf-image-based-upgrade-troubleshooting.adoc[leveloffset=+1]
```

Comment:
This section is not specific to ztp or IBGU. Move it one level up?

Reply:
If you're referring to troubleshooting, it is included in both flows for convenience:
* [Non-gitops](https://81685--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-image-based-upgrade-base#cnf-image-based-upgrade-troubleshooting_cnf-non-gitops)
* [GitOps](https://81685--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade#cnf-image-based-upgrade-troubleshooting_ztp-gitops)

Reply:
Ok, if that's a typical arrangement in our customer doc, we can keep it as is